lisa/sut_orchestrator/azure/features.py

```diff
@@ -669,6 +669,18 @@ def _call_requirement_method(self, method_name: str, capability: Any) -> Any:
                     value.data_disk_size = self._get_disk_size_from_iops(
                         value.data_disk_iops, disk_type_iops
                     )
+        elif method_name == RequirementMethod.intersect:
+            # intersect two int range
+            assert isinstance(self.data_disk_iops, search_space.IntRange)
```

Comment:
It doesn't need to required both of them are not None. If one of them is None, another one should be effective. That's how intersect works with None type.